src/pages/InflectPage/components/Pali1AutoComplete/Pali1AutoComplete.tsx

```diff
@@ -34,15 +34,17 @@ export const Pali1AutoComplete = ({ db, initialValue, onChangePali1 }: Pali1Auto
     }
 
     const loadOptions = () => {
-      const results = db.exec(
-        `SELECT pāli1, pos, ${
-          script === 'Latn' || script === 'xx' ? 'name' : script.toLowerCase()
-        } FROM '_stems' left join '_abbreviations' on pos = name WHERE pāli1 like '${PSC.convertAny(
-          selectedWord.pali1,
-          PSC.Script.RO,
-        )}%' order by pāli1 asc`,
-      )
-      console.log(results)
+      const selectedWordSanitised = selectedWord.pali1.replaceAll("'", '')

```

Comment:
does not sound right on multiple levels
1. we shouldn't mess with the user input - user knows better than the code what they are trying to do
2. what happens to double quotes and frequently used punctuations?

Reply:
1. after a disc. with Bhante B - the cases where punctuations appear are usually copy-pasted from texts. ie no intentional typing in. 
Possible solutions - replace ' with ''(two single quotes) or remove '. 
Reason for choosing removal over replacement : Removing ' (screenshot A) has an advantage compared to replacing with '' (screenshot B) [close results appear]-  
A: ![sA](https://user-images.githubusercontent.com/60284375/114512160-6cb73a80-9c56-11eb-98e7-d016057c0666.png)
B: ![sB](https://user-images.githubusercontent.com/60284375/114512166-6e80fe00-9c56-11eb-852c-62adef29db54.png)

2. list of punctuations tested - !"#$%&'()*+,-./:;<=>?@[\]^_`{|}~. Single quotes(specifically odd numbers of them)  was the only case that failed.

Reply:
ok. if we are removing punctuations - we should remove all punctuations. if not, then remove nothing.

i suggest removing all punctuations as pāli does not have any - above list from python + few more that occur commonly in the texts ie ‘, ’, “,”